website/docs/r/manager_cluster.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+subcategory: "Fabric"
+layout: "nsxt"
+page_title: "NSXT: nsxt_manager_cluster"
+description: A resource to configure an NSXT cluster.
+---
+
+# nsxt_manager_cluster
+
+This resource provides a method for creating an NSXT cluster with several nodes
+This resource is supported with NSX 4.1.0 onwards.
+
```

Comment:
I think we need some details here, mentioning that the main node is specified in provider config

Reply:
Done. Also mentioned in the last few lines that import is not supported for this resource